psalm.xml

```diff
@@ -4,9 +4,12 @@
         phpVersion="8.1"
         resolveFromConfigFile="true"
         rememberPropertyAssignmentsAfterCall="false"
+        ensureOverrideAttribute="false"
+        findUnusedCode="false"
```

Comment:
Please keep this on

Reply:
It's a library, everything is unused code!

Reply:
No, public code must be marked with `@api`, the remaining code is private, unused code.

Reply:
@danog I disagree - in a library everything is api, unless it's marked @internal, or is a private method.
This is littering the code with unnecessary annotations.

Reply:
Please keep this config off unless psalm offers a way to "default-mark" everything non-internal & non-private as API.

Reply:
The problem with default-marking everything non-internal and non-private as API is with non-library codebases (that have no public surface apart from API controllers and other entry points).

Some not so clean library codebases have the same problem as well, where internal details are needlessly exposed as public and not marked as internal.

Explicitly marking public API with `@api` solves both issues.

Reply:
@danog I think it would be a good thing to make that configurable in psalm. `libraryAssumeApi="true"` or such.